### DISCUSSION TITLE
would it make sense to include the version of Bundler used?

### DISCUSSION THREAD

**Discussion by Adnan-Bacic** [react-native-community/discussions-and-proposals#619] [Ideas]
the react-native template has taken steps to include versions of toolsets used, with files such a `Gemfile`(Cocoapods), `.node-version` and `.ruby-version`.

the react-native template, however, does not come with various `.lock` files, such as `yarn.lock`, `Podfile.lock` and `Gemfile.lock`.

if you go to the bottom of the `Gemfile.lock` file, you can see these lines of code:

```
RUBY VERSION
   ruby 2.7.5p203

BUNDLED WITH
   2.3.18
```

we already have the **RUBY VERSION** in the `.ruby-version` file, so we can confirm that this is fine. but we dont have the version of **Bundler**, which is shown under **BUNDLED WITH**. this means that until you run `bundle install` to generate the `Gemfile.lock` file, you wont see the version of **Bundler** you use.

should we worry/care about the version of Bundler used? does it make sense to include it in the project template?

**Comment by kelset** [comment]
not really, we are taking things in a different direction actually by removing the ruby requirements: https://github.com/facebook/react-native/commit/e57b6d11fa39aa96d92a7a6b8c91429974bef81c

we'll have that commit land in all currently supported versions (0.69/0.70/0.71) in the next patch releases and 0.72

**Comment by cortinico** [comment]
> the react-native template has taken steps to include versions of toolsets used, with files such a Gemfile(Cocoapods), .node-version and .ruby-version.

Just a small note to echo what @kelset mentioned: we removed `.node-version` and `.ruby-version` and took a different approach to specify the required Node/Ruby version which is more relaxed and should allow easier configuration.

> should we worry/care about the version of Bundler used? does it make sense to include it in the project template?

Nope, you should not care about the Bundler version used
